### ISSUE TITLE
could not install both sp and dp

### ISSUE THREAD

**Issue by k-okada** [rdiankov/collada-dom#20]
reported at -> http://answers.ros.org/question/222892/collada-dom24-sp-dev-causes-dist-upgrade-to-fail-on-ubuntu-1404/
```
The following NEW packages will be installed:
  collada-dom-dev collada-dom2.4-sp-dev ros-indigo-collada-parser
  ros-indigo-collada-urdf ros-indigo-desktop ros-indigo-desktop-full
  ros-indigo-robot ros-indigo-robot-model ros-indigo-simulators


Unpacking collada-dom2.4-sp-dev (2.4.4.1-ubuntu1~trusty1) ...  dpkg:
error processing archive
/var/cache/apt/archives/collada-dom2.4-sp-dev_2.4.4.1-ubuntu1~trusty1_amd64.deb
(--unpack): trying to overwrite
'/usr/lib/cmake/collada_dom-2.4/collada_dom-config.cmake', which is also
in package collada-dom2.4-dp-dev 2.4.4.0-ubuntu1~trusty1 dpkg-deb:
error: subprocess paste was killed by signal (Broken pipe)
```

**Comment by k-okada** [comment]
done
https://launchpad.net/~openrave/+archive/ubuntu/testing/+sourcepub/5880437/+listing-archive-extra
https://launchpad.net/~openrave/+archive/ubuntu/testing/+sourcepub/5880436/+listing-archive-extra
https://launchpad.net/~openrave/+archive/ubuntu/testing/+sourcepub/5880435/+listing-archive-extra
https://launchpad.net/~openrave/+archive/ubuntu/testing/+sourcepub/5880434/+listing-archive-extra

**Comment by gavanderhoorn** [comment]
Just for my understanding: how was this fixed? I'd expect some related commit(s), but can't find any - either here or at the launchpad project.

**Comment by k-okada** [comment]
I updated cmake fils as #21 and run following commands to push to the ppa
```
 2111  rm -fr *; cmake -DOPT_BUILD_PACKAGES=ON -DOPT_BUILD_PACKAGE_DEFAULT=OFF -DOPT_DOUBLE_PRECISION=OFF         -DDPUT_HOST="ppa:openrave/testing" -DPACKAGE_VERSION="4"         -DCPACK_PACKAGE_CONTACT="OpenRAVE Testing (Testing and Releasing of OpenRAVE Packages) <openrave.testing@gmail.com>"         -DCPACK_DEBIAN_DISTRIBUTION_RELEASES="trusty;utopic;vivid;willy" -DCPACK_DEBIAN_DISTRIBUTION_NAME="Ubuntu" ../collada-dom; make dput
 2112  rm -fr *; cmake -DOPT_BUILD_PACKAGES=ON -DOPT_BUILD_PACKAGE_DEFAULT=ON -DOPT_DOUBLE_PRECISION=ON         -DDPUT_HOST="ppa:openrave/testing" -DPACKAGE_VERSION="4"         -DCPACK_PACKAGE_CONTACT="OpenRAVE Testing (Testing and Releasing of OpenRAVE Packages) <openrave.testing@gmail.com>"         -DCPACK_DEBIAN_DISTRIBUTION_RELEASES="trusty;utopic;vivid;willy" -DCPACK_DEBIAN_DISTRIBUTION_NAME="Ubuntu" ../collada-dom; make dput

```
see 
https://github.com/ros/robot_model/issues/118#issuecomment-166537315 too

**Comment by gavanderhoorn** [comment]
Ah ok. #21 didn't exist yet when I checked the repository. That would've made things obvious.

Thanks

**Comment by k-okada** [comment]
This has been solved as of 31, Dec, 2016, with 2.4.4.4 of collada-dom-dev. Sorry for trouble.
see http://lists.ros.org/lurker/thread/20151225.002725.4ef881c4.en.html for discussion
